packages/iov-bns/src/bnsconnection.spec.ts

```diff
@@ -1190,7 +1184,11 @@ describe("BnsConnection", () => {
     expect(faucetAcct.value()!.balance.length).toEqual(1);
     const end = faucetAcct.value()!.balance[0];
     expect(end).not.toEqual(start);
-    expect(end.whole + 680).toEqual(start.whole);
+    expect(end.quantity).toEqual(
+      Long.fromString(start.quantity)
```

Comment:
@iov/bns already has the package "long" as a dependency (because of protobuf) and bn.js is not available. The numbers is this test are known to fit in the signed 64 bit range of `Long`.

Reply:
In one hand we have the exercise to analyze the max amount of a blockchain, then decide if it fits in specific range (Long, Int53, Uint64), in the other hand, use big number for all the blockchains. What is the advantage of have specific types for different ranges against use bignumber for all, now that we have quantity as string and not as number

Reply:
Yeah, well, we could use a lowest common denominator, which is probably bn.js. But math on amounts is blockchain specific anyway. As long as all the chain implementations are conform to the common interface (integer strings now), the implementation can do what it wants and what is most appropriate given specific requirements and the available types.

In this case both `Long` and `BN` work fine but `Long` is already available.

Reply:
This test is explicitly for BNS, where it is guaranteed 9 fractional digits.
For ethereum, where the 18 digits overflow number, but work in long, yes, it is better to be more careful, as 100 ETH will overflow long.

I think fine as is, but good eye to look at it.